### DISCUSSION TITLE
Remove the drag opition in image

### DISCUSSION THREAD

**Discussion by devimako** [area17/twill#2456] [Q&A]
## Description

I need to remove the drag  option or changing the position  on images in form module is this possible 
<img width="909" alt="Screenshot 2023-02-27 at 1 28 52 PM" src="https://user-images.githubusercontent.com/99637767/221506658-2414576f-0a19-421f-a3fd-87e6de1883ee.png">

### Versions

 Twill version:  2
 Laravel version: 9
 PHP version: 8
 Database engine: pgsql

**Comment by ifox** [comment]
It looks like you are trying to lock the field, which can be done using the disabled prop.
